Extensions/Effects/reflection-pixi-filter.js

```diff
@@ -0,0 +1,68 @@
+gdjs.PixiFiltersTools.registerFilterCreator("Reflection", {
+  makePIXIFilter: function(layer, effectData) {
+    let time = 0;
+
+    const reflectionFilter = new PIXI.filters.ReflectionFilter(
+      effectData.booleanParameters.mirror,
+      effectData.doubleParameters.boundary,
+      [
+        effectData.doubleParameters.amplitudeStart,
+        effectData.doubleParameters.amplitudeEnding
+      ],
+      [
+        effectData.doubleParameters.waveLengthStart,
+        effectData.doubleParameters.waveLengthEnding
+      ],
+      [
+        effectData.doubleParameters.alphaStart,
+        effectData.doubleParameters.alphaEnding
+      ],
+      time
+    );
+
+    return reflectionFilter;
+  },
+  update: function(filter, layer) {
+    if (filter.animated) {
+      filter.time -= layer.getElapsedTime() / 1000;
```

Comment:
Interesting, why subtracting rather than adding the elapsed time? :)

Reply:
Because movement of waves go on up, so i substract it for go down.